app/jobs/notify_track_syncer_about_track_changes_job.rb

```diff
@@ -0,0 +1,13 @@
+class NotifyTrackSyncerAboutTrackChangesJob < ApplicationJob
+  queue_as :default
+
+  def perform(track)
+    body = {
+      event_type: 'track_changes',
+      client_payload: {
+        tracks: ["exercism/#{track.slug}"]
+      }
+    }.to_json
+    Exercism.octokit_client.post("https://api.github.com/repos/exercism/org-wide-files/dispatches", body)
```

Comment:
The only one "downside" of this is that it means the Sidekiq server needs internet access, which it doesn't currently have (and I'd rather it didn't). I wonder we could move into a command rather than a job and run it inline?